Products/CMFPlone/interfaces/controlpanel.py

```diff
@@ -545,7 +545,7 @@ class ITinyMCELayoutSchema(Interface):
             u'Enter a JSON-formatted style format configuration. '
             u'A format is for example the style that get applied when '
             u'you press the bold button inside the editor. '
-            u'See http://www.tinymce.com/wiki.php/Configuration:formats'),
+            u'See https://www.tinymce.com/wiki.php/Configuration:formats'),
```

Comment:
Great, cool, it is just that if I do that jenkins will fail cause length is 85.

Since I do not know enough Python someone else has to do that then.

@hvelarde willing to do so ?